client/src/component/Review.jsx

```diff
@@ -1,29 +1,98 @@
 import React from 'react';
+import styles from '../componentsStyles/ReviewStyle.jsx';
+
 
 class Review extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      reviewsHaveBeenClicked: false,
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event, review) {
+    event.preventDefault();
```

Comment:
try removing preventDefault and see if it still works-- a lot of times you only need it for forms and links